### PR TITLE
Add VS Code

### PR DESCRIPTION
**Pull request by joaomoreno**
This adds Visual Studio Code to the list of projects

### CODE REVIEW

File: _data/projects/vscode.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+tags:
+- vscode
+- visual
+- studio
+- code
+- editor
+- ide
+upforgrabs:
+  name: help wanted
+  link: https://github.com/microsoft/vscode/issues?q=is%3Aopen+is%3Aissue+label%3A"help+wanted"
```
Comment (ritwik12) [inline _data/projects/vscode.yml]: @joaomoreno Please change this to https://github.com/microsoft/vscode/labels/help%20wanted

File: _data/projects/vscode.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+name: Visual Studio Code
+desc: Code editing. Redefined.
```
Comment (ritwik12) [inline _data/projects/vscode.yml]: @joaomoreno It will be better to have a good and explanatory description please.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by joaomoreno** [comment]
@ritwik12 I've addressed the feedback.

**Review by shiftkey — APPROVED** [review]

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@joaomoreno Thanks!!

**Pull request merged**